samples/Retries/Program.cs

```diff
@@ -43,12 +43,13 @@
         BaseDelay = TimeSpan.FromSeconds(1),
 
         // The recommended backoff type for HTTP scenarios
+        // https://www.baeldung.com/resilience4j-backoff-jitter
```

Comment:
This is C# specific, rather than Java.

```suggestion
        // See here for more information: https://github.com/App-vNext/Polly/wiki/Retry-with-jitter#more-complex-jitter
```